packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -768,4 +769,26 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       uri: keepAliveUrl
     });
   }
+
+  /**
+   * Make a network request to send a reaction.
+   * @param {Object} options
+   * @param {Url} options.reactionChannelUrl
+   * @param {Reaction} options.reaction
+   * @param {string} options.senderID
+   * @returns {Promise}
+   */
+  sendReaction({ reactionChannelUrl, reaction, participantId }: { reactionChannelUrl: string, reaction: Reaction, participantId: string }) {
+    const uri = `${reactionChannelUrl}`;
```

Comment:
Is there a reason we're using a template literal here?

Reply:
probably used one of the existing functions as a starting point and didn't remove the template literal (which was useful in the other functions). Will remove.

Reply:
Removed.